packages/docusaurus-theme-classic/src/theme/hooks/useWindowSize.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {useEffect, useState} from 'react';
+
+const desktopThresholdWidth = 996;
+const desktopSize = 'desktop';
+const mobileSize = 'mobile';
+
+function useWindowSize() {
+  const isClient = typeof window === 'object';
```

Comment:
It seems the build fails (check netlify deploy logs), I think you should use `typeof window !== "undefined"` maybe? not sure

Reply:
Looks like ```typeof window !== "undefined"``` is indeed more common.